recipes/BEHST/test.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+cd $PREFIX/bin
+
+./project.sh --test
```

Comment:
A user will not be changing to the bin directory first, so this is not a good test of whether it will actually work in practice.